src/map/status.cpp

```diff
@@ -8981,8 +9021,6 @@ t_tick status_get_sc_def(struct block_list *src, struct block_list *bl, enum sc_
 
 	// Cap minimum rate
 	rate = max(rate, scdb->min_rate);
-	// Cap minimum duration
-	tick = i64max(tick, scdb->min_duration);
```

Comment:
Keep in mind that if you remove this here, then status changes with flag&SCSTART_NOTICKDEF cannot have a minimum duration applied.